awx/main/tasks/system.py

```diff
@@ -482,7 +483,6 @@ def cluster_node_heartbeat():
     for inst in instance_list:
         if inst.hostname == settings.CLUSTER_HOST_ID:
             this_inst = inst
-            instance_list.remove(inst)
```

Comment:
Removing this, I would like to avoid `inst` getting in the `lost_instances` list. By definition, if you are running this code, you are not a lost instance. Any issues will be handled in `this_isnt.local_health_check()` which is a separate block of code. I'm a little worried that the order of operations might jerk its error status back and fourth.

Reply:
Dealt with, for now.  A real cleanup of this code seems in order, though.